corehq/apps/case_importer/exceptions.py

```diff
@@ -112,6 +112,28 @@ class InvalidDate(CaseRowError):
     )
 
 
+class InvalidNumber(CaseRowError):
+    title = gettext_noop('Invalid Number')
```

Comment:
I realize this is following the prior `Error` examples, but any idea why `gettext_noop` is being used here rather than `gettext_lazy`?